### PR TITLE
Fix:#2883: Add Release notes for v1.20

### LINKED ISSUE #2883: Release Notes for 1.20
- Add default browser capability (https://github.com/brave/brave-ios/issues/2785)
- Update user agent (https://github.com/brave/brave-ios/issues/2864)

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

This pull request fixes #2883 

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`

## Test Plan:

NA

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.20](https://github.com/brave/brave-ios/releases/tag/v1.20)
+
+ - Implemented capability to set Brave as default browser. ([#2785](https://github.com/brave/brave-ios/issues/2785))
```
Comment (kjozwiak) [inline CHANGELOG.md]: I think using `Added ability to set Brave as the default browser` might sound a bit better/easier to understand. I don't think there's a need to use the word `capability` as that's a given due as it's being implemented/added.

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Comment by MrWaste** [comment]
Please Fix this.
Proof:

Brave
Duck

**Comment by kjozwiak** [comment]
@MrWaste this issue is for the release notes for the next release. Please create a new issue describing either the issue you're having or a feature request via https://github.com/brave/brave-ios/issues/new/choose. However, if you're referring to the missing `Default Browser` setting in Brave, that will be available one the above version (1.20) goes out soon 👍

**Comment by kjozwiak** [comment]
@srirambv we'll need to add the following into the release notes as well as it's happening on the current release:

* Youtube failing to load next videos #2810 - https://github.com/brave/brave-ios/issues/2810

@diracdeltas should the following security issues be mentioned in the release notes? They're currently set as `release-notes/exclude`:

* another security issue in serialize-javascript #2855 - https://github.com/brave/brave-ios/issues/2855
* Fix Security Token for YoutubeAdblock.js #2813 - https://github.com/brave/brave-ios/issues/2813

@bsclifton @anthonypkeane believe we should probably include the following as well as these changes are user facing. General rule is that if we've changed/fixed/added anything new that the user will see, we need to mention it. Obviously sometimes there's exceptions. Any reason why these are being labeled as `release-notes/exclude`? Is there a reason why we shouldn't be adding them?

* IAP VPN Subscription Text #2747 - https://github.com/brave/brave-ios/issues/2747 <-- `UI text change`
* VPN text update #2825 - https://github.com/brave/brave-ios/issues/2825 <-- another user facing text change
* Rewards text update #2829 - https://github.com/brave/brave-ios/issues/2829 <-- large text change
* Update onboarding flow #2832 - https://github.com/brave/brave-ios/issues/2832 <-- probably important to mention IMO

**Comment by underling3311** [comment]
@kjozwiak do you have a time frame when the update will be released?

**Comment by bsclifton** [comment]
@MrWaste unfortunately not - we submitted 1.20 back on Friday... and we've been waiting to hear back since then. We've had a few rejections from the App Store before this latest attempt- with each attempt, we're addressing items. The rejections we received (for Guideline violations) weren't for anything new, they were for features we have had for quite some time (roughly a year)

**Comment by underling3311** [comment]
@bsclifton
Hopefully Epic can stop most the of the Bs from the apps being rejected.. I do some app development too. Good Luck!

**Comment by underling3311** [comment]
@bsclifton

I was just wondering, what kind of features got it declined. You said they were there for 1 year ish. Hm.. 
Is there is post somewhere there's the rejections from the appstore.

**Comment by gh0sti** [comment]
> @MrWaste unfortunately not - we submitted 1.20 back on Friday... and we've been waiting to hear back since then. We've had a few rejections from the App Store before this latest attempt- with each attempt, we're addressing items. The rejections we received (for Guideline violations) weren't for anything new, they were for features we have had for quite some time (roughly a year)

I think its BS for Apple to treat you guys differently from Edge, Chrome, and Firefox. All of them got their default browser option update out with iOS 14 release. Even duckduckgo did. Apple doesn't treat each developer the same.

**Comment by srirambv** [comment]
@kjozwiak ready for re-review

**Comment by underling3311** [comment]
How is the update with apple rejecting the app updates?
Everyday I am checking for update and still nothing. I can't for this update to finally fix the YouTube problem + pus default browser.

**Comment by iccub** [comment]
We just got approved :)

**Review by iccub — APPROVED** [review]

**Review by iccub — APPROVED** [review]

**Review by bsclifton — APPROVED** [review]
++

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**